### ISSUE TITLE
unsinged git tag in daily-composer

### ISSUE THREAD

**Issue by emanuelb** [phpmyadmin/scripts#10]
in "scripts/website/daily-composer"
```
    # Create tag
git tag -a -m "Composer tag for $tag" $newtag
```
which will create unsigned tag:
https://github.com/phpmyadmin/composer/releases/tag/4.7.0

**Comment by nijel** [comment]
That's true - it's created automatically from the cron job, so the signing here doesn't add that much value...

**Comment by emanuelb** [comment]
it does (if signed by bot account) as if signing used it avoid depending on Github SSL connection (as they / MITM can't sign the repo) as it daily it better be signed by bot and not user...
